### ISSUE TITLE
Orbit.from_sbdb raises unhelpful error if no object was found

### ISSUE THREAD

**Issue by astrojuanlu** [poliastro/poliastro#916]
```python
In [9]: Orbit.from_sbdb("67/P")
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
<ipython-input-9-75d3da3ffd57> in <module>
----> 1 Orbit.from_sbdb("67/P")

~/.pyenv/versions/poliastro37_4/lib/python3.7/site-packages/poliastro/twobody/orbit.py in from_sbdb(cls, name, **kargs)
    572             )
    573 
--> 574         a = obj["orbit"]["elements"]["a"].to(u.AU) * u.AU
    575         ecc = float(obj["orbit"]["elements"]["e"]) * u.one
    576         inc = obj["orbit"]["elements"]["i"].to(u.deg) * u.deg

KeyError: 'orbit'
```

It should give something more meaningful, like `ValueError("Object 67/P not found")`

**Comment by priyanshurohilla** [comment]
I am on it

**Comment by priyanshurohilla** [comment]
@iamabhishek0 I was working on it.:)

**Comment by astrojuanlu** [comment]
@priyanshurohilla [As per my comment from 15 days ago](https://github.com/poliastro/poliastro/issues/798#issuecomment-609019954):

> On the other hand, it gives the impression that others should refrain to work on the issue, which is wrong.

I will be more explicit this time: blocking issues this way is not the correct attitude. Please refrain from doing so in the future.

**Comment by priyanshurohilla** [comment]
> @priyanshurohilla [As per my comment from 15 days ago](https://github.com/poliastro/poliastro/issues/798#issuecomment-609019954):
> 
> > On the other hand, it gives the impression that others should refrain to work on the issue, which is wrong.
> 
> I will be more explicit this time: blocking issues this way is not the correct attitude. Please refrain from doing so in the future.

@astrojuanlu  understood.. Won't happen from now onwards 😊

**Comment by astrojuanlu** [comment]
Fixed in #919!
